Unity-Technologies/Graphics
# Fixed asset preview being rendered white because of static lighting sky.

# Purpose of this PR

Fixed an issue where the asset preview could be rendered over exposed if the current scene had a static lighting sky with high exposure https://fogbugz.unity3d.com/f/cases/1237824/
Fixed an issue where static lighting would not update when removing the static lighting sky profile in the lighting panel.

# Testing status
## Manual Tests
Tested both fixes manually in the provided scene.
@Unity-Technologies/gfx-qa-hdrp as an additional test, please make sure that the static sky still works as it should (changing profile, type of sky etc, with ambient probe / lightmap). I did some testing but this part is pretty sensitive so I'd like another person testing as well.
